source/about/mattermost-v10-changelog.md

```diff
@@ -3,6 +3,109 @@
 ```{Important}
 ```{include} common-esr-support-upgrade.md
 ```
+(release-v10.10-feature-release)=
+## Release v10.10 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: July 16, 2025**
+
+### Important Upgrade Notes
+ - Added a new column ``DefaultCategoryName`` to the ``Channels`` table. This is nullable and stores a category name to be added/created when new users join a channel. This is only used if the ``ExperimentalChannelCategorySetting`` is enabled. No database downtime is expected for this upgrade. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for more details.
+
+```{Important}
+If you upgrade from a release earlier than v10.9, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls plugin [v1.9.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.9.1).
+ - Pre-packaged GitLab plugin [v1.10.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.10.0).
+ - Pre-packaged GitHub plugin [v2.4.0](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.4.0).
+ - Pre-packaged Boards plugin [v9.1.3](https://github.com/mattermost/mattermost-plugin-boards/releases/tag/v9.1.3).
```

Comment:
This should be v9.1.4 once the cherry pick is complete https://github.com/mattermost/mattermost/pull/31771 

cc// @Rajat-Dabade 

```suggestion
 - Pre-packaged Boards plugin [v9.1.4](https://github.com/mattermost/mattermost-plugin-boards/releases/tag/v9.1.4).
```